### ISSUE TITLE
Issue on page /content/ioos_installation_conda.html

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/ioos_code_lab#150]
We should add info about installing git on a local machine. Especially installing when user doesn't have admin privilege's. I'm working on that now and will document my findings for Windows.

**Comment by ocefpaf** [comment]
While one can `conda install git` that would require to always use the conda prompt and activate the env where they installed it, unless they install on the base env.

The other option is git-bash. In a way the first step is similar b/c one must always start from the git-bash.

**Comment by MathewBiddle** [comment]
* browse to https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
* find installing on windows and go to link https://git-scm.com/download/win
* download the latest 64 bit version to `programs/`
![image](https://github.com/ioos/ioos_code_lab/assets/8480023/e6eac822-01b0-447b-a128-1fa0ea3bbf50)
* run application
* install to ~/programs/Git

**Comment by MathewBiddle** [comment]
I think it would be beneficial to include installing the standalone git-bash. I've walked a few folks through these instructions and git is something I think we're missing.

**Comment by MathewBiddle** [comment]
My recommendation would be to add a sentence in the Creating conda environment section that says something along the lines of "While the environment is being created, install git following the instructions at https://git-scm.com/book/en/v2/Getting-Started-Installing-Git".

**Comment by ocefpaf** [comment]
We got some feedback from git-bash users and conda git in conda-forge. The main complain is that they have two new "terminal icons" one for the AnacondaPrompt and another one for the git-bash, and they still have the vanilla one from Windows. That together confusing activation scripts for each terminal, make installing git from conda at the base environment, an easier option. They users may miss the git functionalities but modern Windows does have the Linux subsystem, making git-bash redundant.

What do you think? Do you use git-bash + conda env regularly on Windows?

**Comment by MathewBiddle** [comment]
Yes, agreed on the multiple terminal windows. I added shortcuts to my taskbar for all the different terminals I use and I've simply memorized which one to use when. 

![image](https://github.com/ioos/ioos_code_lab/assets/8480023/8e0fc7d4-f32a-4bfe-9c44-46a60fa88f77)

My Windows machine does not have the Linux subsystem (or it's not enabled) so I need git-bash if I want a bash like shell.

Only rarely do I use git-bash to do git stuff. Mainly I use pycharm's git integration to manage all those actions through the GUI. But, I think git needs to be installed to make that work.


Conda environment question. If a user installs git from conda in the base environment, would git be available to all other environments?

**Comment by ocefpaf** [comment]
> would git be available to all other environments?

Yes. Not in the vanilla terminal though.
